src/oadp_resources/oadp/datamover/volume_snapshot_restore.py

```diff
@@ -1,6 +1,49 @@
 from ocp_resources.resource import NamespacedResource
+
+from oadp_constants.oadp.datamover.volume_snapshot_restore import VolumeSnapshotRestorePhase
 from oadp_constants.resources import ApiGroups
 
 
-class VolumeSnapshotLocation(NamespacedResource):
+class VolumeSnapshotRestore(NamespacedResource):
     api_group = ApiGroups.DATAMOVER_OADP_API_GROUP.value
+
+    def is_vsr_done(self):
```

Comment:
why not is_done?

Reply:
changed:
https://github.com/oadp-qe/openshift-adp-python-wrapper/compare/de92a4e33e60946201b177dab723c5a9acc7446e..3a77300c3cf18aca8c155a20d3f19c2766a4adbe

https://github.com/oadp-qe/openshift-adp-python-wrapper/compare/3a77300c3cf18aca8c155a20d3f19c2766a4adbe..7823980fa166eaced3f8a8dcf2c8d915a53bd936